### ISSUE TITLE
Is it really converging? 

### ISSUE THREAD

**Issue by ahnHeejune** [GeorgeSeif/AlexNet-TensorFlow#4]
I ran this code with minor changes 
  - cost = tf.reduce_mean(tf.nn.softmax_cross_entropy_with_logits(logits=pred, labels=y)) ## FIX BUG
  - batch_size =  16   # due to the GPU memory limit in my Laptop (GTX 940M)

But it still accuracy is just 0.3 ~ 0.4 after 100000 iterations.

Can you show the results loss graph for me, to check what is problem.
